### DISCUSSION TITLE
Feature requests

### DISCUSSION THREAD

**Discussion by Mystfit** [Mystfit/Unreal-StableDiffusionTools#11] [Ideas]
I'm trying to think of some new features for the plugin.

- Masked inpainting using layers/stencil masks.
- Outpainting with 360 image support.
- Projection map generated images back onto the world.
- Assign projection mapped textures to nodes in space. Blend between nodes depending on distance.
- Outpainting using projection mapped world textures.
- Midas depth texture generation.
- Feed generated frames back into the movie pipeline. Blend between the input image and the last generated frame (advected forwards using motion vectors).
- Bake Audio Synesthesisa NRT values into sequencer curves to drive prompt weights.

**Comment by NGC38** [comment]
This is amazing stuff you are creating... Though, too early than what everyone was expecting.
It's no doubt the future AI applications together with Unreal Engine.
I've just a serious recommendation. By what I saw so far, users will have some difficulties on understanding the real applications of this plugin. In my opinion, we are not yet there for animation production.
Yet, knowing the actual capacities of Stable Diffusion on generating 360 degrees panorama images (with the help of some LORA models and specific prompt words), we can easily turn the resulting images into HDRI textures that, with the backdrop plugin (built-in Unreal) can simply fill a dome/sphere on any scenario generated in UE5 projects.
If so, I believe that if you focalize your plugin for automatizing the process of generating the HDRI textures (which seems pretty simple in programming), artists and developers can use your plugin to generate unlimited fantastic 360 degrees scenarios with a click of a mouse.
Will soon join you on Discord where we can chat about this (username: Sculvus).
